CMSIS/Core/Include/core_cm4.h

```diff
@@ -1108,12 +1108,36 @@ typedef struct
 #define TPI_FFSR_FlInProg_Msk              (0x1UL /*<< TPI_FFSR_FlInProg_Pos*/)        /*!< TPI FFSR: FlInProg Mask */
 
 /* TPI Formatter and Flush Control Register Definitions */
-#define TPI_FFCR_TrigIn_Pos                 8U                                         /*!< TPI FFCR: TrigIn Position */
```

Comment:
Ok, but then why not documenting them properly to promote them to "official" status ? Are they really "deprecated" ?

Reply:
Its a matter of how to keep a common style. You added `#define TPI_FFCR_TrigIN_Pos                 8U` instead, i.e. upper case N instead of lower case one. Do you prefer to keep both? In this case we should agree on one official version. Or should we simply stay with the lower case version and drop your new one?

Reply:
I put those in CAPITAL because it is the way they are declared in http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0337e/BABIFBCI.html table 13.8 (direct link http://infocenter.arm.com/help/topic/com.arm.doc.ddi0337e/BABIFBCI.html#BIIJEJFB doesn't work) hence I wanted to stay as close as possible to the official documentation (for what it worth, considering the numerous divergences)

Reply:
That makes totally sense to me. But removing the non-capital form might break existing implementations. Thats why I think we might simply keep both for a while.

Reply:
Sounds reasonable enough to me.

Reply:
May I ask you to update the PR accordingly? I'll merge it asap.

Reply:
Done...